core/pom.xml

```diff
@@ -249,19 +246,38 @@
                 <artifactId>jaxb2-maven-plugin</artifactId>
                 <executions>
                     <execution>
+                        <id>schemagen</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>schemagen</goal>
```

Comment:
I remember last time we had a discussion about using Lombok in model classes, to remove those boilerplate getters, setters and toStrings, but after adding Lombok the build was failing. I created a mojohaus/jaxb2-maven-plugin#88 to address this, but did not get any response yet. 

But, I was able to add Lombok and build the project with previous major version of the jaxb2 plugin (they rewrote a lot in 2x). In my opinion, clean code in core is more important that using the latest maven plugin version, thus I would recommend to apply this solution. 

In order to build with lombok, I changed the `jaxb2-maven-plugin`version to `1.6` and its execution configuration to the following:
```
                     <execution>
                        <id>schemagen</id>
                        <phase>generate-resources</phase>
                        <goals>
                            <goal>schemagen</goal>
                        </goals>
                        <configuration>
                            <workDirectory>${project.build.directory}/generated-resources/schemagen</workDirectory>
                            <includes>
                                org/dozer/builder/model/jaxb/**
                            </includes>
                            <transformSchemas>
                                <transformSchema>
                                    <uri>http://dozermapper.github.io/schema/bean-mapping</uri>
                                    <toFile>bean-mapping.xsd</toFile>
                                </transformSchema>
                            </transformSchemas>
                        </configuration>
                    </execution>
                    <execution>
                        <id>xjc</id>
                        <phase>generate-test-sources</phase>
                        <goals>
                            <goal>xjc</goal>
                        </goals>
                        <configuration>
                            <schemaDirectory>
                                ${project.basedir}/src/test/xsd/jaxb
                            </schemaDirectory>
                            <packageName>org.dozer.vo.jaxb.employee</packageName>
                            <outputDirectory>target/jaxb-sources</outputDirectory>
                        </configuration>
                    </execution>
``` 
 
After that the plugin generated the XSD from classes with Lombok annotations applied.

Reply:
Done, will start doing the lombok changes when i get some more time